### PR TITLE
changes made

### PR DESCRIPTION
**Pull request by abrolnalin**

### CODE REVIEW

File: plivo/resources/powerpacks.py [hunk 1]
```diff
@@ -129,6 +129,19 @@ def add_number(self,  number):
...
+    @validate_args(
+    tollfree=[of_type(six.text_type)]
+    )
+    def add_tollfree(self,  tollfree):
+        numberpool_uuid = self.get_numberpooluuid()
+        if numberpool_uuid != "":
+            return self.client.request(
+            'POST', ('NumberPool',numberpool_uuid,'Tollfree', tollfree),
+            response_type=None,
+            objects_type=Powerpack)
```
Comment (narayana-plivo) [inline plivo/resources/powerpacks.py]: why object type is Powerpack here?

File: plivo/resources/numberpools.py [hunk 2]
```diff
@@ -175,3 +177,68 @@ def find(self, shortcode):
...
+        return self.client.request('GET', ('NumberPool',self.number_pool_id,'Tollfree', tollfree),
+            response_type=None,
+            objects_type=None)
+
+    @validate_args(
+    tollfree=[of_type(six.text_type)]
+    )
+    def remove(self, tollfree, unrent=False):
+        params = {}
+        print('fasfas', tollfree)
```
Comment (narayana-plivo) [inline plivo/resources/numberpools.py]: why do you need print statement here?

File: tests/resources/test_powerpacks.py [hunk 3]
```diff
@@ -86,13 +86,37 @@ def test_add_number(self):
...
             self.client.current_request.url)
 
+    @with_response(200)
+    def test_add_tollfree(self):
+        powerpack = self.client.powerpacks.get(uuid='d35f2e82-d387-427f-8594-6fa07613c43a')
+        response = powerpack.add_number( tollfree='18772209942')
+        #response= powerpack.numberpool.numbers.add( number='18772209942')
+        self.assertEqual('POST', self.client.current_request.method)
+        self.assertUrlEqual(
+            'https://api.plivo.com/v1/Account/MAXXXXXXXXXXXXXXXXXX/NumberPool/ca5fd1f2-26c0-43e9-a7e4-0dc426e9dd2f/Tollfree/18772209942/',
```
Comment (narayana-plivo) [inline tests/resources/test_powerpacks.py]: if the powerpack not found case, is this will pass?

File: tests/resources/test_powerpacks.py [hunk 4]
```diff
@@ -86,13 +86,37 @@ def test_add_number(self):
...
         response= powerpack.remove_number( number='15799140336')
         #response= powerpack.numberpool.numbers.remove( number='15799140336')
         self.assertEqual('DELETE', self.client.current_request.method)
 
+    @with_response(200)
+    def test_remove_tollfree(self):
+        powerpack = self.client.powerpacks.get(uuid='d35f2e82-d387-427f-8594-6fa07613c43a')
+        response= powerpack.remove_tollfree( tollfree='18772209942')
+        #response= powerpack.numberpool.tollfree.remove( number='18772209942')
+        self.assertEqual('DELETE', self.client.current_request.method)
```
Comment (narayana-plivo) [inline tests/resources/test_powerpacks.py]: can you do assert on delete status code?

File: tests/resources/test_powerpacks.py [hunk 5]
```diff
@@ -86,13 +86,37 @@ def test_add_number(self):
...
+            self.client.current_request.url)
+
     @with_response(200)
     def test_remove_number(self):
         powerpack = self.client.powerpacks.get(uuid='d35f2e82-d387-427f-8594-6fa07613c43a')
         response= powerpack.remove_number( number='15799140336')
         #response= powerpack.numberpool.numbers.remove( number='15799140336')
         self.assertEqual('DELETE', self.client.current_request.method)
 
+    @with_response(200)
```
Comment (narayana-plivo) [inline tests/resources/test_powerpacks.py]: on remove status code is 200?

File: tests/resources/test_powerpacks.py [hunk 6]
```diff
@@ -86,13 +86,37 @@ def test_add_number(self):
...
+    def test_remove_tollfree(self):
+        powerpack = self.client.powerpacks.get(uuid='d35f2e82-d387-427f-8594-6fa07613c43a')
+        response= powerpack.remove_tollfree( tollfree='18772209942')
+        #response= powerpack.numberpool.tollfree.remove( number='18772209942')
+        self.assertEqual('DELETE', self.client.current_request.method)
+
+    @with_response(200)
```
Comment (narayana-plivo) [inline tests/resources/test_powerpacks.py]: same here?

File: tests/resources/test_powerpacks.py (hunk 5 above)
Comment (abrolnalin) [reply tests/resources/test_powerpacks.py]: Yes for remove number/tollfree/shortcode apis return 200

File: tests/resources/test_powerpacks.py (hunk 6 above)
Comment (abrolnalin) [reply tests/resources/test_powerpacks.py]: same as above

**Pull request merged**